cmd/gitops-server/cmd/cmd.go

```diff
@@ -187,7 +191,12 @@ func runCmd(cmd *cobra.Command, args []string) error {
 
 	ctx := context.Background()
 
-	cl, err := cluster.NewSingleCluster(cluster.DefaultCluster, rest, scheme, cluster.DefaultKubeConfigOptions...)
+	oidcPrefixes := kube.UserPrefixes{
+		UsernamePrefix: options.OIDC.UsernamePrefix,
+		GroupsPrefix:   options.OIDC.GroupsPrefix,
+	}
```

Comment:
This should also incorportate authServer.AuthConfig.OIDCConfig.GroupsPrefix/UsernamePrefix I think?

Given https://github.com/weaveworks/weave-gitops/blob/cb8585c028ca6287220c49e9c3643d60f64a645d/pkg/server/auth/init.go#L40-L41

We should merge them w/ the secret taking higher precedence and show a similar logging message.

Reply:
Got it 👍 Thank you 🙂